scripts/tls_test.c

```diff
@@ -37,8 +37,27 @@
 static void usage(FILE *out, char *cmd)
 {
 	fprintf(out, "usage:\n");
-	fprintf(out, "  %s --connect <address> --port <port> [--key <key] [--cert <file>]+ [--times <n>]\n", cmd);
-	fprintf(out, "  %s --listen <address> --port <port> --key <key> [--cert <file>]+ [--times <n>]\n", cmd);
+	fprintf(out, "  %s --connect <address> --port <port> [--key <key] [--cert <file>] [--cacert <file>]+ [--times <n>]\n", cmd);
+	fprintf(out, "  %s --listen <address> --port <port> --key <key> --cert <file> [--cacert <file>]+ [--times <n>]\n", cmd);
+	fprintf(out, "\n");
+	fprintf(out, "options:\n");
+	fprintf(out, "    --help                   print help and exit\n");
```

Comment:
Not sure, but you might have overlooked the "or" in my comment about this :) With the "options:" separator, using two spaces indentation should be perfectly fine (and even provides two characters more for the description - I might adapt the other tools to this too because using that much indentation there seems wasteful). The four space indentation was a suggestion in case the options were separated by an empty line only, so they can be better distinguished from the usage lines that are indented by two spaces.